README.md

```diff
@@ -1,6 +1,6 @@
 # Moby Ryuk
 
-This project helps you to remove containers/networks/volumes by given filter after specified delay.
+This project helps you to remove containers/networks/volumes/images by given filter after specified delay.
```

Comment:
unrelated change, accidentally committed another contribution 😅 

the image pruning algorithm remains the same